pkg/envoy/lds/connection_manager.go

```diff
@@ -15,11 +15,20 @@ import (
 	"github.com/openservicemesh/osm/pkg/featureflags"
 )
 
+// ConnManagerFilterDirection defines, for filter terms, the direction of the traffic from an application
+// perspective, in which the connection manager filters will be applied
+type ConnManagerFilterDirection string
+
 const (
 	statPrefix = "http"
+
+	// Incoming defines in-mesh inbound and ingress traffic driections
+	Incoming = "incoming"
+	// Outgoing defines in-mesh outbound and egress traffic directions
+	Outgoing = "outgoing"
```

Comment:
nit: use `inbound, outbound` to be consistent with other parts of the code (ex. RDS route direction).

Also un-export the const.

Reply:
I'll talk to you offline, had a Q about this.